internal/blocktx/bcnet/mcast/listner.go

```diff
@@ -0,0 +1,87 @@
+package mcast
+
+import (
+	"context"
+	"errors"
+	"log/slog"
+	"os"
+
+	"github.com/bitcoin-sv/arc/internal/blocktx/bcnet"
+	"github.com/bitcoin-sv/arc/internal/blocktx/store"
+	"github.com/bitcoin-sv/arc/internal/multicast"
+	"github.com/libsv/go-p2p/wire"
+)
+
+var ErrUnableToCastWireMessage = errors.New("unable to cast wire.Message to blockchain.BlockMessage")
+
+var _ multicast.MessageHandlerI = (*Listner)(nil)
+
+type Listner struct {
```

Comment:
Typo, in filename as well
```suggestion
type Listener struct {
```